components/admin_console/billing/billing_subscriptions/cloud_trial_banner.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import moment from 'moment';
+
+import {getBrowserTimezone} from 'utils/timezone';
+
+import useOpenSalesLink from 'components/common/hooks/useOpenSalesLink';
+import AlertBanner from 'components/alert_banner';
+import UpgradeLink from 'components/widgets/links/upgrade_link';
+
+import './cloud_trial_banner.scss';
+
+export interface Props {
+    trialEndDate: number;
+}
+
+const CloudTrialBanner = ({trialEndDate}: Props): JSX.Element | null => {
+    const endDate = new Date(trialEndDate);
+    const {formatMessage} = useIntl();
+    const openSalesLink = useOpenSalesLink();
+
+    if (trialEndDate === 0) {
+        return null;
+    }
+
+    return (
+        <AlertBanner
+            mode={'info'}
```

Comment:
Per the design it looks like this should be dismiss-able?

Reply:
you are right @julmondragon, thanks!. @marianunez @annelieseklein once the admin dismisses this banner, should it be dismissed just for the session or should be dismissed for the remaining duration of the trial (meaning dismissed for good)?

Reply:
@johndavidlugtu and I are reconvening on this to make a final decision, will update shortly

Reply:
@pvev it should be dismissed/snoozed for 10 days just for that admin